embeddable-say-editor/app/components/simple-editor.js

```diff
@@ -537,4 +553,18 @@ export default class SimpleEditorComponent extends Component {
     const { marks } = setup;
     marks.redacted = redacted;
   }
+
+  setupRdfaEditor(setup) {
```

Comment:
The tools need the sidebar, so we should have `setup.uiConfig.sidebar = true;`, in case e.g. it's the only plugin they want to activate.

Reply:
I'm dumb

Reply:
Done